docs/about.md.bak

```diff
@@ -0,0 +1,8 @@
+# About UO Studio
```

Comment:
I think this .bak file needs to go.

Would you mind adding it to `.gitignore`?

Reply:
Didn't I remove it already? lol

I will modify the `.gitignore` file for future changes.

Reply:
Done!